diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6a29924..1dea2a1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,10 +5,6 @@ option(BUILD_DEBUG "Include debugging symbols in library and print some usefull
 option(DYNAMIC_ID "Generate a dynamic stack ID to distinguish calls." ON)
 option(OPENMP_MPI "Generate graphviz files for exclusively running functions." OFF)
 
-set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/common)
-
-find_package(Scorep REQUIRED)
-
 set(PLUGIN_SOURCE_FILES ${PROJECT_SOURCE_DIR}/src/graphviz.c)
 
 add_definitions("-Wall -Wextra -pedantic -std=c11")
@@ -25,5 +21,5 @@ if(BUILD_DEBUG)
 	add_definitions("-ggdb")
 endif()
 
-include_directories(${PROJECT_SOURCE_DIR}/src/ ${SCOREP_INCLUDE_DIRS})
++include_directories(${PROJECT_SOURCE_DIR}/src/ ${PROJECT_SOURCE_DIR}/build/scorep_substrate_graphviz/include/)
add_library(${PROJECT_NAME} SHARED ${PLUGIN_SOURCE_FILES})
